xstate-chatbot/nodejs/src/machine/vitals.js

```diff
@@ -730,7 +1054,15 @@ const vitalsFlow = {
         }
       }
     },
+    noUserFound: {
+      id: 'noUserFound',
+      onEntry: assign((context, event) => {
+        dialog.sendMessage(context, dialog.get_message(messages.noUserFound, context.user.locale), false);
+      }),
+      always: ''
```

Comment:
⚠️ This will put user in null state. So user will be stuck at that place forever and will not be able to communicate with the chatbot. Please don't write this transition anywhere ⚠️

Reply:
removed noUserFound